apps/backend/db_patches/0187_AddExperimentSafetyReviewTemplate.sql

```diff
@@ -0,0 +1,53 @@
+DO
+$$
+DECLARE
+    exp_safety_review_template_id_var int;
+    exp_safety_review_topic_id_var int;
+    questionary_id_var int;
+BEGIN
+    IF register_patch('AddExperimentSafetyReviewTemplate.sql', 'Yoganandan Pandiyan', 'Add Experiment Safety review template', '2025-03-03') THEN
```

Comment:
We may need to update the dates do not know if they will affect how database patches are applied.